packages/react-native-sdk/src/components/call/CallControls/AcceptCallButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCall } from '@stream-io/video-react-bindings';
+import React, { useCallback } from 'react';
+import { CallControlsButton } from './CallControlsButton';
+import { theme } from '../../../theme';
+import { Phone } from '../../../icons';
+
+/**
+ * The props for the Accept Call button.
+ */
+type AcceptCallButtonProps = {
+  /**
+   * Handler to be called when the accept call button is pressed.
+   * @returns void
+   */
+  onPressHandler?: () => void;
+  /**
+   *  Handler to be called after the incoming call is accepted.
+   */
+  onAcceptHandler?: () => void;
```

Comment:
this usecase seems new to me.. two handlers and one for after and one for complete

can we remove the `onAcceptHandler` here

Reply:
React has it, that is why I added it for uniformity. Integrator can display there logic on what should happen after accepting the call, without overriding the entire handler.